backend/services/goals_service.py

```
@@ -231,3 +238,116 @@ def check_and_populate_goals(self, user_id: int):
             self.populate_monthly_goals(user_id)
 
         logger.info("New goals populated successfully")
+
+    def initialize_user_goals(self, user_id: int):
```

Ruff (0.8.2)

Missing return type annotation for public function `initialize_user_goals`

Add return type annotation: `None`

(ANN201)